bucket/xnviewmp.json

```
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://www.xnview.com/en/xnviewmp/",
+    "description": "Photo viewer, image resizer & batch converter.",
+    "version": "0.93.1",
+    "license": "Freeware",

```

```suggestion
    "license": "Freeware|Proprietary",
```
as license.txt says:
```
2. License

XnViewMP is provided as freeware for private (non commercial), or educational use, including non-profit organization (i.e. schools, universities, public authorities, police, fire brigade, and hospitals). 

In this case, you are granted the right to use and to make an unlimited number of copies of this software.

Company must purchase licenses to be able to use XnViewMP.
```